### PR TITLE
Updated UI for User Authentication

### PR DESCRIPTION
**Pull request by aniyahb**
I've updated the UI for the Login and SignUp page to include the header: 
Screenshot 2024-06-26 at 7 39 15 PM

Screenshot 2024-06-26 at 7 39 23 PM

### CODE REVIEW

File: frontend/src/LogInSignUp/LogInSignUp.jsx [hunk 1]
```diff
@@ -33,7 +35,9 @@ const LogInSignUp = () =>{
                     <input type="password" placeholder="Password" />
                 </div>
 
-                {action==="Sign Up"?<div></div>:<div className="ForgotPassword">Forgot Password <span>Click Here!</span></div>}
+                {action==="Sign Up"?
+                <div></div>:
```
Comment (KatsiarynaMashokha) [inline frontend/src/LogInSignUp/LogInSignUp.jsx]: are you planning to add smth else into this `<div>` element?

Comment (aniyahb) [reply frontend/src/LogInSignUp/LogInSignUp.jsx]: No

Comment (KatsiarynaMashokha) [reply frontend/src/LogInSignUp/LogInSignUp.jsx]: so in this case you can use `&&` operator which would render en element if the condition on the left side evaluates to true, so you can do smth like: `{action !== "Sign Up" && <div className="ForgotPassword">Forgot Password <span>Click Here!</span></div>}` so you are not rendering an empty `div`
Here is more info on `&&` operator: https://react.dev/learn/conditional-rendering (under `Logical AND operator (&&) `)

Comment (alishahimtiaz) [reply frontend/src/LogInSignUp/LogInSignUp.jsx]: LGTM, thank you!

**Pull request merged**